src/recipes.jl

```diff
@@ -1117,6 +1123,12 @@ end
         plotattributes[:y], plotattributes[:x], plotattributes[:z] =
             error_coords(yerr, y, x, z)
     end
+    for i in [:x, :y, :z]
+        _scale= Symbol(i, :scale)
+        if _scale ∈ keys(plotattributes) && plotattributes[_scale] == :log10
```

Comment:
This applies also for other log scales and likewise we'd need to consider the case of x-errors with log-scale

Reply:
`i` will iterate over `:x, :y, :z` and clamp each of their error bar in the same fashion, so this should be fine?

Reply:
I am just saying that the `:xerror` recipe probably also needs this fix